add warp to l2 cheatcode + add rollup address to node info

rahul-kothari opened the pull request:
Close #1275 - you can now change time on L2
Also add rollup address to node info

rahul-kothari committed:
add warp to l2 cheatcode

LHerskind commented on yarn-project/end-to-end/src/fixtures/cheat_codes.ts:
```diff
@@ -256,6 +256,19 @@ export class L2CheatCodes {
+   * @param to - The timestamp to set the next block to (must be greater than current time)
+   * @param rollupContract - The rollup contract address
+   */
+  public async warp(to: number, rollupContract: EthAddress): Promise<void> {
```
It should be possible for us to get the `rollupAddress` from the node instead of passing it in.

LHerskind committed:
feat: add rollup address to node info (#1483)

Adds a getter of the rollup address and fixes some cheat-code work

# Checklist:
Remove the checklist to signal you've completed it. Enable auto-merge if
the PR is ready to merge.
- [ ] If the pull request requires a cryptography review (e.g.
cryptographic algorithm implementations) I have added the 'crypto' tag.
- [ ] I have reviewed my diff in github, line by line and removed
unexpected formatting changes, testing logs, or commented-out code.
- [ ] Every change is related to the PR description.
- [ ] I have
[linked](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
this pull request to relevant issues (if any exist).

rahul-kothari committed:
add checks on warp

Maddiaa0 reviewed (approved):
lgtm
